src/main/app/src/Components/Landscape/Search/Search.tsx

```diff
@@ -63,7 +63,7 @@ const Search: React.FC<PropsInterface> = ({locateFunction, setSidebarContent, ..
   const search = useCallback(
     (searchTerm: string, identifier: string) => {
       if (searchTerm.length < 2) return;
-
+      setHasChange(false);
```

Comment:
Wheter it is here or after the `get()` should not really matter - does it?

Reply:
Yes, here it prevents further requests because the ui is rendered several times before the response is available.